modules/proc_installing-the-rollout-manager-plugin.adoc

```diff
@@ -0,0 +1,70 @@
+:_mod-docs-content-type: PROCEDURE
```

Comment:
This file can be removed from the PR: the next release of OpenShift GitOps (e.g. v1.13) will ensure that the plugin is automatically installed for the user.

Reply:
Fixed.